hack/debug.sh

```diff
@@ -542,16 +542,29 @@ case "$1" in
     "")
 	if [ -z "$aos_master_service" ]; then
 	    echo "Usage:"
-	    echo "  [from master]"
+	    echo "  [From master]"
+	    echo "  Automatic:"
 	    echo "    $0"
 	    echo "  Gathers data on the master and then connects to each node via ssh"
 	    echo ""
-	    echo "  [from any other machine]"
+	    echo "  Manual:"
+	    echo "    $0 --master"
+	    echo "  Runs only master debugging on local machine"
+	    echo ""
+	    echo "  [From any node]"
+	    echo "  Automatic:"
 	    echo "    $0 MASTER-NAME"
```

Comment:
It's not "from any node", it's from literally *any* machine other than the master. Actually, it's almost certainly *not* going to be a node, since if your nodes can ssh to the master then the master can almost certainly ssh to the nodes as well, and so you might as well have just run it from the master in the first place.

This is for when the master doesn't have ssh keys to let it connect to nodes, and so instead, you run it from your desktop machine, where you do have an ssh key that lets you log in to both the master and the nodes.